makefile.cargo

```diff
@@ -11,26 +11,30 @@ CONFIGURE_FLAGS := \
 
 ifneq ($(HOST),$(TARGET))
 
-CC ?= $(TARGET)-gcc
-CPP ?= $(TARGET)-gcc -E
-CXX ?= $(TARGET)-g++
-AR ?= $(TARGET)-ar
-
-CONFIGURE_FLAGS += --target=$(TARGET) --disable-gold
+	ifeq (armv7-linux-androideabi,$(TARGET))
+		# Fix: armv7 TARGET used by rust is not the same as the TARGET needed for CXX toolchain.
```

Comment:
What does the `Fix:` prefix here mean? I first interpreted it as FIXME, but I don't think that's correct.